src/test/scala/org/broadinstitute/dsde/firecloud/service/ServiceSpec.scala

```diff
@@ -15,11 +17,22 @@ trait ServiceSpec extends FreeSpec with ScalaFutures with ScalatestRouteTest wit
 
   implicit val routeTestTimeout = RouteTestTimeout(5.seconds)
 
+  // OPTIONS is not included because we accept all OPTIONS requests to all endpoints for CORS.
```

Comment:
I don't think this is true... according to David, those requests are handled by the proxy and never even hit Scala code

Reply:
What's the implication, remove `OPTIONS` from `allHttpMethods`?

Reply:
Yup, remove `OPTIONS` from `allHttpMethods` and remove the comment

Reply:
Check with David to be sure, though

Reply:
@davidangb Does Adam's suggestion sound good to you, too?

Reply:
agree - remove `OPTIONS` from the list of `allHttpMethods`, no need to test it here. (plus, removing it from the list makes it consistent with the comment)